tests/test.py

```diff
@@ -6823,24 +6823,129 @@ def executeBulkInsert(self, query, values):
 
 
 class TestSearchableTransparentEncryption(BaseSearchableTransparentEncryption):
```

Comment:
Do we want to add a couple of tests for not-equal operator for `SearchableTokenization`?

Reply:
makes sense. added